x-pack/platform/plugins/shared/fleet/server/routes/epm/file_handler.ts

```diff
@@ -17,10 +17,30 @@ import { getAsset } from '../../services/epm/archive/storage';
 import { getBundledPackageByPkgKey } from '../../services/epm/packages/bundled_packages';
 import { pkgToPkgKey } from '../../services/epm/registry';
 import { unpackArchiveEntriesIntoMemory } from '../../services/epm/archive';
+import { FleetError } from '../../errors';
 
 const CACHE_CONTROL_10_MINUTES_HEADER: HttpResponseOptions['headers'] = {
   'cache-control': 'max-age=600',
 };
+
+const ALLOWED_MIME_TYPES = [
+  'image/svg+xml',
+  'image/jpeg',
+  'image/png',
+  'image/gif',
+  'application/json',
+  'application/yaml',
+  'text/plain',
+  'text/markdown',
+  'text/yaml',
```

Comment:
Just re-adding my previous comment, as it was outdated. I see you added the `application/yaml` type.
Still curious if using or updating this constant make sense.
https://github.com/elastic/kibana/blob/c4a80ad3bbf2d777a8ac973c5ed2c6bd1c9700a5/x-pack/platform/plugins/shared/cases/common/constants/mime_types.ts#L90

Reply:
Looks like markdown is not in that list, and we will need that one.
it may be easier to keep a smaller list here of what we support specifically for packages. wdyt?

Reply:
That is fine with me. I just wanted to make sure we considered using a similar approach. Thanks!